### ISSUE TITLE
 No raw posts (<article> elements) were found in this page.

### ISSUE THREAD

**Issue by eviscerat** [kevinzg/facebook-scraper#877]
I use scraper to scrape a public fb page. Since yesterday I get the following when running the script:
`No raw posts (<article> elements) were found in this page.`
` Page parser did not find next page URL`

**Comment by FlorenciaBCabrera** [comment]
Hola @eviscerat pudiste solucionarlo? tengo el mismo problema.

Hello @eviscerat could you solve it? I have the same problem.

**Comment by ikbal-Benahmed** [comment]
I have the same problem, can anyone help ?

**Comment by eviscerat** [comment]
@FlorenciaBCabrera Nope, I did not resolve it, I didn't find the time to investigate and solve it myself, I just stopped using it altogether,

**Comment by basheerpaliyathu** [comment]
@kevinzg i am also getting same issue since last 5 days. 
tried different cookies, used different ips to request, changed user agents 

- any suggestion?

**Comment by eabanoz** [comment]
There are 187 comments and 15 shares for this post https://www.facebook.com/100064371026715/posts/847913440697682 . When I use the facebook scraper I get this error message:
WARNING:facebook_scraper.facebook_scraper:No raw posts (<article> elements) were found in this page. When I use the git+https://github.com/moda20/facebook-scraper.git@master version, I get some 20 comments and bunch of error messages like these:
ERROR:facebook_scraper.extractors:Unable to parse comment <Element 'div' class=('eu',) id='comment_replies_more_1:847913440697682_285865264522157'>: 'NoneType' object has no attribute 'text'
ERROR:facebook_scraper.extractors:Unable to parse comment <Element 'div' class=('bv', 'ev', 'ew', 'ex')>: 'NoneType' object has no attribute 'text'
ERROR:facebook_scraper.extractors:Unable to parse comment <Element 'div' class=('el',)>: 'NoneType' object has no attribute 'text'
ERROR:facebook_scraper.extractors:Unable to parse comment <Element 'div' class=('em',)>: 'NoneType' object has no attribute 'text'
ERROR:facebook_scraper.extractors:Unable to parse comment <Element 'div' class=('bv', 'en', 'eb')>: 'NoneType' object has no attribute 'text'
